Gemfile.lock

```diff
@@ -2,71 +2,131 @@ GEM
   remote: http://rubygems.org/
```

Comment:
It isn't required.  I guess it's just an argument about whether we want to pin the website tightly to a set of versions (either via Gemfile.lock or a version pin in Gemfile) so we don't get any surprises when the website is built.

That said, the PR seems to demonstrate that the stability and compatibility of the stack between 2013 and 2015 is excellent.  It's likely that GitHub would advertise if there was an upcoming incompatible change.